src/assets/quasar-logo-full.svg

```diff
@@ -0,0 +1,191 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Do we need a Quasar logo?

Reply:
As I've stated above, the project was created using **quasar-cli** which bundles some default files.